Doc/library/struct.rst

```diff
@@ -267,12 +267,26 @@ platform-dependent.
 | ``P``  | :c:expr:`void \*`        | integer            |                | \(5)       |
 +--------+--------------------------+--------------------+----------------+------------+
 
+Additionally, if IEC 60559 compatible complex arithmetic (Annex G) is
```

Comment:
This should probably link to the specification.

Reply:
Which link do you suggest?  Wikipedia page for IEEE 754, C11 standard?

Reply:
Probably the C11 standard.